### DISCUSSION TITLE
List of the CSV files used for the conversion

### DISCUSSION THREAD

**Discussion by 1JKW1** [aramis-lab/clinica#990] [Q&A]
Hi，When I do adni-to-bids, I can't find the required csv file, without carefully stating what the required csv file contains.
For example 
For all modalities: ADNIMERGE.csv
T1-weighted MRI: MPRAGEMETA.csv, MRIQUALITY.csv, MAYOADIRL_MRI_IMAGEQC_12_08_15.csv

**Comment by ghisvail** [comment]
I don't understand your issue. Could you provide some basic information, such as:

What did you do? Which command did you run? What is your expected outcome?

**Reply by 1JKW1** [reply]
Thank you for your reply.
First , according to the Clinica Documentation , the list of CSV files for conversion is mentioned in the appendix of adni-to-bids,
where it is mentioned that the file ADNIMERGE.csv is needed for all modalities,
for T1-weighted MRI the files MPRAGEMETA.csv ,  MRIQUALITY.csv ,  MAYOADIRL_ MRI_IMAGEQC_12_08_15.csv are needed
However , in accordance with the names of these four CSV files, I didn't find these four CSV files on adni website, and I don't know what's contained in these four CSV files.
Second , when i was runing the command : clinica convert adni-to-bids [OPTIONS] DATASET_DIRECTORY CLINICAL_DATA_DIRECTORY BIDS_DIRECTORY    
the  outcome is : FileNotFoundError: [Errno 2] No such file or directory: '/test/clinical_date/MPRAGEMETA.csv'

**Comment by ghisvail** [comment]
> I didn't find these four CSV files on adni website

Did you follow the [download section](https://aramislab.paris.inria.fr/clinica/docs/public/latest/Converters/ADNI2BIDS/#downloading-adni) of our documentation?

**Reply by 1JKW1** [reply]
Thanks again for your reply!
I need to read the documentation again.Thanks

**Reply by xupin262** [reply]
Hi,

I meet the same issue. ADNI doesn't provide MPRAGEMETA.csv (MRIQUALITY.csv 和 MAYOADIRL_MRI_IMAGEQC_12_08_15.csv) any more after October 2024. Can you happened to download those file share it? I appreciate it.

Best,
Pin Xu

**Reply by acse-sw5223** [reply]
Hi,

We have the same issue. As ADNI no longer provides some required datasets, we were unable to run the Clinica data converter successfully. Can anybody who has the CSV files share them?

Best,
Shiyun Wa
